### DISCUSSION TITLE
Welcome to axiom Discussions!

### DISCUSSION THREAD

**Discussion by pry0cc** [pry0cc/axiom#216] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hahwul** [comment]
Cool! That's what I wanted. Thank you so much @pry0cc and @0xtavian :D
